webknossos/webknossos/annotation/annotation.py

```diff
@@ -92,11 +93,20 @@ def binary(self) -> Union[bytes, BinaryIO]:
         else:
             return open(self.file, "rb")
 
+    def get_volume_layer_names(self) -> Iterable[str]:
+
+        return (volume._get_name_or_id() for volume in self._nml.volumes)
+
     def save_volume_annotation(
         self,
         dataset: Dataset,
         layer_name: str = "volume_annotation",
         largest_segment_id: Optional[int] = None,
+        # If the annotation contains multiple volume layers,
+        # source_volume_name has to be provided.
+        # Use Annotation.get_volume_layer_names() to look up
+        # available layers.
```

Comment:
Please add this or sth. similar to the docstring instead, so it shows up in the docs